docs/source/API/alphabetical.rst

```diff
@@ -54,6 +54,9 @@ Containers
 Core
 ----
 
+.. |SubviewType| replace:: Subview
+.. _SubviewType: core/view/Subview_type.html
```

Comment:
please write acomment to explain why this fixes it and your thinking behind this solution